src/js/rx/components/Breadcrumbs.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+
+class Breadcrumbs extends React.Component {
+  render() {
+    const { pathname } = this.props.location;
+    const { prescription } = this.props;
+
+    const crumbs = [
+      <a href="/" key="home">Home</a>,
+      <a href="/healthcare" key="healthcare">Health Care</a>,
+    ];
+
+    if (pathname.match(/\/\d+$/)) {
+      crumbs.push(<a href="/healthcare/prescriptions" key="prescriptions">Prescription Refills</a>);
```

Comment:
@U-DON I thought about this and the only benefit that `Link` provides is that it is router-aware. In this case, our [base name](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/rx/rx-entry.jsx#L17) is `/healthcare/prescriptions` so all of the paths are relative to that.

I'll change it to use `Link` but we will need to override the paths by directly setting the `href` attribute on `Link`. (see latest commit for details).